docs/assemblies/common_configurations.adoc

```diff
@@ -69,6 +69,7 @@ ifeval::["{build}" != "downstream"]
         edpm_ovn_bgp_agent_local_ovn_northd_image: "quay.io/podified-antelope-centos9/openstack-ovn-northd
         edpm_ovn_bgp_agent_local_ovn_controller_image: "quay.io/podified-antelope-centos9/openstack-ovn-controller
         edpm_telemetry_node_exporter_image: quay.io/prometheus/node-exporter
+        edpm_telemetry_kepler_image: "quay.io/sustainable_computing_io/kepler"
```

Comment:
nit: I know this is docs, and maybe most of the problem isn't on this patch, but the mismatched quotes caught my eye. The node exporter line is unquoted, but the kepler line has "double-quotes", so I looked up/down to compare to the others for consistency and see it's a mix of unquoted (our images only!) and the rest of the lines have a leading quote with no terminating one. Either the unterminated double-quote is a feature of adoc (or yaml?) that I'm unaware of (in which case your line is wrong), or most of these lines are wrong, and ours are additionally the odd ones out again.

A more general comment is that if the docs are reproducing a large chunk of strings that come from the code, it might be worthwhile to auto-generate them so they remain correct and pastable for our users.

I guess this is not a request for a change on this patch so much as a sanity-check; are all these unterminated quotes intentional? If they are not, I suggest changing them all to be unquoted instead.

Reply:
Part of the docs is already generated. The one involving kustomize that is. We could, in principle, write a script to extract certain chunks of our code and turn it into adoc. That being said, I'd much rather use an existing solution if there is one.

As for the inconsistency, yep, that's a recurring issue. I suggest we merge this change, and later submit a PR fixing the docs. Including it here would be messy.